lib/salus/scanners/pattern_search.rb

```diff
@@ -60,7 +61,9 @@ def run
             ".",
             *(match_exclude_directory_flags || global_exclude_directory_flags),
             *(match_exclude_extension_flags || global_exclude_extension_flags),
-            *(match_include_extension_flags || global_include_extension_flags)
+            *(match_include_extension_flags || global_include_extension_flags),
+            "--blocksize",
```

Comment:
Be sure to document in docs/scanners/pattern_search.md as part of this PR

Also adding support for err-show-line-length and err-skip-line-length might be a nice win here.

```
['err-show-line-length', 'err-skip-line-length'].each do |flag|
	command_array << "--#{flag }"if match.key?(flag')
end

```
That would allow users of Salus to not only set their desired threshold via blocksize but allow them to disable the error should the linelength exceed blocksize.